app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontAddProductsWithCustomOptionToWishlistTest.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontAddProductsWithCustomOptionToWishlistTest">
+        <annotations>
+            <features value="Wishlist"/>
+            <stories value="Add product with customizable option to wishlist"/>
+            <title value="Add product with customizable option to wishlist."/>
+            <description value="Add product with custom radio button option to wishlist."/>
+            <group value="wishlist"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-34245"/>
+        </annotations>
+        <before>
+            <createData entity="SimpleSubCategory" stepKey="categoryFirst"/>
+            <createData entity="SimpleProduct" stepKey="simpleProduct">
+                <requiredEntity createDataKey="categoryFirst"/>
+            </createData>
+
+            <updateData entity="productWithOptionRadiobutton" createDataKey="simpleProduct" stepKey="updateProductWithOptions">
+                <requiredEntity createDataKey="simpleProduct"/>
+            </updateData>
+
+            <createData entity="Simple_US_Customer" stepKey="customer"/>
+            <magentoCLI command="indexer:reindex" stepKey="reindex"/>
+            <magentoCLI command="cache:flush" stepKey="flushCache"/>
+        </before>
+        <after>
+            <deleteData createDataKey="simpleProduct" stepKey="deletesimpleProduct"/>
+            <deleteData createDataKey="categoryFirst" stepKey="deleteCategoryFirst"/>
+            <magentoCLI command="indexer:reindex" stepKey="reindex"/>
+            <magentoCLI command="cache:flush" stepKey="flushCache"/>
```

Comment:
Is there a possibility to clean the particular cache types instead of wiping out the entire cache? 
The point is, we do it twice in the same test and add the test performance degradation in that way not only to the current test but to the following tests that will use the cache as well.